docs/jwst/jump/arguments.rst

```diff
@@ -56,8 +56,28 @@ The ``jump`` step has five optional arguments that can be set by the user:
 
 * ``-min_sat_area``:  The minimum number of saturated pixels required to meet "sat_required_snowball".
 
-* ``--expand_factor``: A multiplicative factor applied to the enclosing circle or ellipse. This larger area will have all pixels flagged as having a jump.
+* ``--expand_factor``: A multiplicative factor applied to the enclosing ellipse for snowballs. This larger area will have all pixels flagged as having a jump.
 
-* ``--use_ellipses``:  A boolean value that if True will cause minimum enclosing ellipses to be fit to the jump contour rather than the default circles. This should be set to True for MIRI observations.
+* ``--use_ellipses``:  deprecated
 
 * ``--sat_required_snowball``: A boolean value that if True requires that there are saturated pixels within the enclosed jump circle.
+
+* ``--min_sat_radius_extend``: The minimum radius of the saturated core of a snowball required to for the radius of the saturated core to be extended.
+
+*``--sat-expand``: Number of pixels to add to the radius of the saturated core of snowballs
```

Comment:
Does this argument name really have a dash/hyphen in the middle, or is it an underscore? All the others use underscores.

Reply:
changed to underscore